### ISSUE TITLE
Hide/unhide default layers directly from toolbar (instead of Layers tab)

### ISSUE THREAD

**Issue by SAASTN** [locationtech/jts#649]
It's nice to be able to hide/unhide layers when Input or Inspect layers are showing for cases when geometries cover each other.

**Comment by dr-jts** [comment]
Agreed, and I have thought about this.  Will see if I can work it in.

**Comment by SAASTN** [comment]
Thanks @dr-jts , I suggest these icons. Sorry that cannot help more.
![Layers_ShowHide](https://user-images.githubusercontent.com/47451967/102144158-433dfc00-3e7a-11eb-985a-dba97482fb59.png)
under WTFPL
